jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/ctl/Async.java

```diff
@@ -18,13 +18,13 @@
 
 package org.apache.jena.fuseki.ctl;
 
-import org.apache.http.HttpHeaders;
 import org.apache.jena.atlas.json.JsonBuilder;
 import org.apache.jena.atlas.json.JsonValue;
 import org.apache.jena.fuseki.async.AsyncPool;
 import org.apache.jena.fuseki.async.AsyncTask;
 import org.apache.jena.fuseki.server.DataService;
 import org.apache.jena.fuseki.servlets.HttpAction;
+import org.apache.jena.riot.web.HttpNames;
```

Comment:
Does `java.net.http` not have an equivalent set of constants to Apache HTTP Client?

Reply:
The source code is available :smile:

If it does, it is not in the jena.net.http package which is the only module export.
Even aside from modules, the rest of the code is in `jdk.internal`.